internal/weldr/api.go

```diff
@@ -1030,21 +1066,29 @@ func (api *API) blueprintsNewHandler(writer http.ResponseWriter, request *http.R
 	}
 
 	contentType := request.Header["Content-Type"]
-	if len(contentType) != 1 || contentType[0] != "application/json" {
+	if len(contentType) != 1 {
```

Comment:
Would it make sense to make a helper out of this patch, that takes a `request` and decodes it into a generic type?